client/resource_group/controller/controller.go

```diff
@@ -213,22 +221,61 @@ func (c *ResourceGroupsController) Start(ctx context.Context) {
 			stateUpdateTicker = time.NewTicker(time.Millisecond * 100)
 		})
 
-		_, revision, err := c.provider.LoadResourceGroups(ctx)
+		_, metaRevision, err := c.provider.LoadResourceGroups(ctx)
+		if err != nil {
+			log.Warn("load resource group revision failed", zap.Error(err))
+		}
+		_, cfgRevision, err := c.provider.LoadGlobalConfig(ctx, nil, controllerConfigPath)
```

Comment:
I've observed that there are two different approaches to interact etcd. do we need to unify it?
- `LoadGlobalConfig` is impled by pd client
- `Watch` is impled by metaClient